domains/client/domain-operator/src/tests.rs

```diff
@@ -8800,3 +8800,134 @@ fn proof_to_tx(
+            None,
+            None,
+            None,
+            false,
+            None,
+        )
+        .expect("EVM runtime call must succeed")
+        .expect("EVM call info must succeed")
+        .value;
+
+    let minimum_transfer_amount =
+        sp_core::U256::from_big_endian(&minimum_transfer_amount_call_result[0..32]);
+    // In evm-domain-test-runtime MinimumTransfer is 1
+    assert_eq!(
+        minimum_transfer_amount,
+        sp_core::U256::from(1u64),
+        "minimum_transfer_amount() mismatch"
+    );
+
+    // Fund an EVM account to transfer from and fetch gas price/nonce
+    let sender_account = address_build(123);
+    let sender_evm_address = sender_account.address;
+    let fund_tx = alice.construct_extrinsic(
+        alice.account_nonce(),
+        pallet_balances::Call::transfer_allow_death {
+            dest: sender_evm_address.0.into(),
+            value: FUNDING_AMOUNT,
+        },
+    );
+    alice.send_extrinsic(fund_tx).await.expect("funding failed");
+
+    let receiver_account_id = Sr25519Keyring::Bob.to_account_id();
+    let receiver_balance_before = ferdie.free_balance(receiver_account_id.clone());
+
+    produce_blocks!(ferdie, alice, 1).await.unwrap();
+
+    let gas_price = alice
+        .client
+        .runtime_api()
+        .gas_price(alice.client.info().best_hash)
+        .unwrap();
+    let sender_account_before = alice
+        .client
+        .runtime_api()
+        .account_basic(alice.client.info().best_hash, sender_evm_address)
+        .unwrap();
+    let sender_nonce = sender_account_before.nonce;
+    let sender_balance_before = sender_account_before.balance;
+
+    // Build calldata for transfer_to_consensus_v1(bytes32,u256)
+    let to = H160::from_low_u64_be(2048);
+    let receiver_ss58 = receiver_account_id.encode();
+    let mut receiver_bytes32 = [0u8; 32];
+    receiver_bytes32.copy_from_slice(&receiver_ss58);
+    let receiver_h256 = sp_core::H256::from_slice(&receiver_bytes32);
+    let transfer_amount = U256::from(TRANSFER_AMOUNT);
+
+    let mut call_data =
+        sp_core::hashing::keccak_256(b"transfer_to_consensus_v1(bytes32,uint256)")[0..4].to_vec();
+    call_data.extend_from_slice(receiver_h256.as_bytes());
+    call_data.extend_from_slice(&transfer_amount.to_big_endian());
+
+    // Create EIP-1559 tx that calls precompile
+    let evm_tx = domain_test_utils::test_ethereum::generate_eip1559_tx::<TestRuntime>(
+        sender_account.clone(),
+        sender_nonce,
+        ethereum::TransactionAction::Call(to),
+        call_data,
+        gas_price,
+    );
+    let eth_ex = generate_eth_domain_sc_extrinsic(evm_tx);
+    alice
+        .send_extrinsic(eth_ex)
+        .await
+        .expect("Failed to send evm tx to transporter precompile");
+
+    // Wait until receiver succesfully receives funds
+    produce_blocks_until!(ferdie, alice, {
+        ferdie.free_balance(receiver_account_id.clone()) > receiver_balance_before
```

Comment:
we should actually check for equality here since we know how much receiver is going to get